.github/workflows/deployProd.yml

```diff
@@ -103,10 +119,13 @@ jobs:
       - uses: azure/login@v1
         with:
           creds: ${{ secrets.AZURE_CREDENTIALS }}
-      - name: Retrieve frontend build
-        uses: actions/download-artifact@v2
-        with:
-          name: frontend-tarball
+      - name: Retrieve frontend build from release
+        run: |
+          curl -L -v \
```

Comment:
Did you mean to leave the `-v` in?

Reply:
Honestly I don’t remember if I did or not—I copied it from my sandbox repo and may or may not have noticed when I did that. It’s the only way to get diagnostic information if something goes wrong, so I’m somewhat inclined to leave it in for troubleshooting purposes? Not strongly committed to it, though.